ui/.build/src/esbuild.ts

```diff
@@ -81,6 +81,9 @@ function esbuildMessage(msg: es.Message, error = false) {
     ? `:${msg.location.line}`
     : '' + (msg.location?.column ? `:${msg.location.column}` : '');
   const srcText = msg.location?.lineText;
+  if (env.prod && error) {
```

Comment:
do we want it to error on dev as well so we can catch the error before pushing live? (and maybe even local?) 
same as below...

Reply:
it will be logged in the console so you will see it appear.
This is used to ensure the step of the workflow fails

Reply:
sorry misunderstood your comment, the build is alrways invoked with prod = true in our CI

Reply:
so for any env, the error still is displayed the same as before (so you will see it in local)
for our CI I make sure the step fails so it's easier to see there was a problem.

Reply:
got it, I misunderstood the env trigger.

Reply:
I think we do not want to throw an error as there might be several errors and we display only the first in the list
We also do not want to have to relaunch watch mode in local because something is temporarily not working properly